101-machine-learning-create/azuredeploy.json

```diff
@@ -1,132 +1,114 @@
+            "kind": "web",
+            "properties": {
+                "Application_Type": "web"
+            }
+        },
+        {
+            "type": "Microsoft.ContainerRegistry/registries",
+            "apiVersion": "2019-05-01",
+            "name": "[variables('containerRegistryName')]",
+            "location": "[if(or(equals(parameters('location'),'australiaeast'),equals(parameters('location'),'brazilsouth'),equals(parameters('location'),'canadacentral'),equals(parameters('location'),'centralus'),equals(parameters('location'),'eastasia'),equals(parameters('location'),'eastus'),equals(parameters('location'),'eastus2'),equals(parameters('location'),'francecentral'),equals(parameters('location'),'japaneast'),equals(parameters('location'),'koreacentral'),equals(parameters('location'),'northcentralus'),equals(parameters('location'),'northeurope'),equals(parameters('location'),'southeastasia'),equals(parameters('location'),'southcentralus'),equals(parameters('location'),'uksouth'),equals(parameters('location'),'westcentralus'),equals(parameters('location'),'westus'),equals(parameters('location'),'westus2'),equals(parameters('location'),'westeurope')),parameters('location'),'southcentralus')]",
+            
+            "sku": {
+                "name": "Standard"
+            },
+            "properties": {
+                "adminUserEnabled": true
+            }
+        },
+        {
+            "type": "Microsoft.MachineLearningServices/workspaces",
+            "apiVersion": "2019-11-01",
+            "name": "[parameters('workspaceName')]",
+            "location": "[if(or(equals(parameters('location'),'australiaeast'),equals(parameters('location'),'brazilsouth'),equals(parameters('location'),'canadacentral'),equals(parameters('location'),'centralus'),equals(parameters('location'),'eastasia'),equals(parameters('location'),'eastus'),equals(parameters('location'),'eastus2'),equals(parameters('location'),'francecentral'),equals(parameters('location'),'japaneast'),equals(parameters('location'),'koreacentral'),equals(parameters('location'),'northcentralus'),equals(parameters('location'),'northeurope'),equals(parameters('location'),'southeastasia'),equals(parameters('location'),'southcentralus'),equals(parameters('location'),'uksouth'),equals(parameters('location'),'westcentralus'),equals(parameters('location'),'westus'),equals(parameters('location'),'westus2'),equals(parameters('location'),'westeurope')),parameters('location'),'southcentralus')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName'))]",
+                "[resourceId('Microsoft.KeyVault/vaults', variables('keyVaultName'))]",
+                "[resourceId('Microsoft.Insights/components', variables('applicationInsightsName'))]",
+                "[resourceId('Microsoft.ContainerRegistry/registries', variables('containerRegistryName'))]"
+            ],
+            "identity": {
```

Comment:
how is this used?

Reply:
The resource workspace isn't available for all regions, so I treated using the condition on the location.

Reply:
I'm asking about the MSI - how is it used, it's never referenced in this template...  If it's not used, remove it.